chapters/fa/chapter2/2.mdx

```diff
@@ -405,9 +405,9 @@ tf.Tensor(
 </div>
 
 
-حالا می‌توانیم ببینیم که پیش‌بینی مدل برای جمله اول `[0.9995,  0.0005]` و برای جمله دوم `[0.9995,  0.0005]` است. این‌ها مقادیر آشنای آماری هستند.
+حالا می‌ببینیم که پیش‌بینی مدل برای جمله اول `[0.9995,  0.0005]` و برای جمله دوم `[0.9995,  0.0005]` است. این‌ها مقادیر آشنای آماری هستند.
 
-برای تبدیل این مقادیر به عنوان دسته تشخیص داده شده می‌توانیم از ویژگی `id2label` تنظیمات مدل استفاده کنیم(در بخش بعدی بیشتر در این مورد صحبت خواهیم کرد):
+برای تبدیل این مقادیر به نام دسته تشخیص داده شده می‌توانیم از ویژگی `id2label` تنظیمات مدل استفاده کنیم(در بخش بعدی بیشتر در این مورد صحبت خواهیم کرد):
```

Comment:
Fixes & suggestions:

- There must be a space before parentheses openings.
- As per previous point regarding a proper equivalent for "label": "برچسب دسته‌بندی" or "لیبل گروه" instead of "نام دسته".

Reply:
گروه doesnt work bc we need to be consistent.

Went with برچسب دسته‌